### ISSUE TITLE
needs sound.

### ISSUE THREAD

**Issue by nedzen** [matthewsimo/plank-workout#1]
can't watch the screen while exercising. It should have a sound to mark exercise change.

**Comment by matthewsimo** [comment]
Good idea, I'll work on it. Had something implemented but iOS support was lack luster. I'm sure there is a shim somewhere since I've started playing with this. Stay tuned.
